PWGCF/FemtoDream/Core/femtoDreamTrackSelection.h

```diff
@@ -146,7 +145,7 @@ class FemtoDreamTrackSelection : public FemtoDreamObjectSelection<float, femtoDr
   /// \tparam T Data type of the track
   /// \param track Track
   /// \return Whether the most open combination of all selection criteria is fulfilled
-  template <typename T>
+  template <class T>
```

Comment:
Strange that this needed to be changed just now. In principle `typename` is larger than `class` embracing it
I guess if local compilation succeeded there should not be any issue with it

Reply:
AFAIK, the `typename` and `class` keywords are equivalent in the context of template arguments. What do you mean by "`typename` is larger than `class`"?